packages/eslint-plugin-next/lib/rules/next-script-for-ga.js

```diff
@@ -0,0 +1,60 @@
+const NodeAttributes = require('../utils/nodeAttributes.js')
+
+const GA_URL = 'www.google-analytics.com/analytics.js'
+const ERROR_MSG =
+  'Use the Script component for loading third party scripts. See: https://nextjs.org/docs/messages/next-script-for-ga.'
```

Comment:
Nit: remove the period so that it doesn't affect the link direct.

```suggestion
  'Use the Script component for loading third party scripts. See: https://nextjs.org/docs/messages/next-script-for-ga'
```